ADOPTERS.md

```diff
@@ -2,103 +2,103 @@
 
 _If you're using Backstage in your organization, please try to add your company name to this list. This really helps the project to gain momentum and credibility. It's a small contribution back to the project with a big impact._
 
-| Organization                                                                | Contact                                                                                                                                                                                                      | Description of Use                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
you should probably just revert this entire file (something like `git fetch origin/master` + `git checkout origin/master ADOPTERS.md` and checking that in)